package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "broccoli-funnel": "^1.2.0",
-    "ember-cli-babel": "^6.3.0",
+    "ember-cli-babel": "^6.9.2",
```

Comment:
Any reason why we’re bumping to 6.9.2?

Reply:
@ynnoj Ah, no that is the latest. But it need to be 6.8.? I think. I can change it to that version.

Reply:
@ynnoj Fixed to the minimum needed version of ember-cli-babel according to
https://emberjs.com/blog/2017/10/11/ember-2-16-released.html

Reply:
The modules API landed in [2.16](https://emberjs.com/blog/2017/10/11/ember-2-16-released.html#toc_changes-in-ember-js-2-16).  Looking at the [output](https://github.com/ember-cli/ember-addon-output/blob/v2.16.2/package.json#L21) of a new `ember-cli@2.16.2` addon, its expected at `6.6.0`. We're at `2.16.2` on master now.

Reply:
Ok, I can change to 6.6.0. I only changed because of this quote from the 2.16 release blog linked above. "Upgrade your ember-cli-babel dependency to v6.8.0 or greater. This will permit your addon to use the new modules in its addon/ and test/ directories."

Reply:
@ynnoj set to version 6.6.0. Hope it still works in an app, I could not test. But the addon tests run fine. =)